# Homework 3

*Name: Yuxuan Han*

*Net ID: yxh204*

---

In [3]:
import numpy as np
import pandas as pd
from scipy import stats

In [4]:
data = pd.read_csv("https://raw.github.com/kevinhan0/PUI2016_fb55/master/Lab3_fb55/times.txt", header = None)
data.columns = ["Time"]

---

### Question 3

A new bus route for line X8 is implemented. MTA wants to know if it improves commute time (travel time at peak hours).

They know what the mean travel time used to be, and measure the new travel time 100 times. Given $T\sim N(\mu=36, \sigma=6)$, is the new bus route improving commute?

Let $H_0$ be the null hypothesis that the average commute time during peak hours stays the same before and after the implementation of the new bus route, and let $H_1$ be the alternative hypothesis that the average commute time during peak hours has improved after the implementation of the new bus route, that is,

$H_0: \mu \leq 36$

$H_a: \mu > 36$

We test our hypothesis using the Z-test for mean, $Z = \dfrac{\bar{x}-\mu_0}{\sigma/\sqrt N}$, at a significance level of $\alpha=0.05$.

In [9]:
mu_0 = 36; sigma = 6
x_bar = np.mean(data.Time)
n = len(data.Time)

In [17]:
t_stat = (x_bar - mu_0)/(sigma/np.sqrt(n))

In [18]:
pval = stats.norm.sf(np.abs(t_stat))
print('z-statistic = %6.3f p-value = %6.4f' % (t_stat, pval))

z-statistic = -2.556 p-value = 0.0053


Since $p=0.0053<\alpha=0.04$, we reject the null hypothesis and accept the alternative hypothesis that the average commute time during peak hours has improved after the implementation of the new bus route.

---